src/SDKs/DataMigration/Management.DataMigration/Microsoft.Azure.Management.DataMigration.csproj

```diff
@@ -17,6 +17,9 @@
 	</PropertyGroup>
 	<PropertyGroup>
 		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+		<AssemblyVersion>0.4.0.0</AssemblyVersion>
```

Comment:
Please remove these lines
`AssemblyVersion` and `AssemblyFileVersion` must be set in the `AssemblyFile.cs` file similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs)

Reply:
Done.